light-model-client/src/commonMain/kotlin/org/modelix/client/light/LightModelClient.kt

```diff
@@ -40,6 +80,13 @@ class LightModelClient internal constructor(
     private var exceptions: MutableList<ExceptionData> = ArrayList()
     private var currentModelQuery: ModelQuery? = null
     private var unappliedQuery: ModelQuery? = null
+    private val modelQLClient: ModelQLClient? = when (connection) {
+        is WebsocketConnection -> ModelQLClient(
+            connection.url.replace(Regex("ws://(.*)/ws"), "http://$1/query"),
```

Comment:
Suggestion: Do not use a regex. Add fields `host` and `port` in `WebsocketConnection` and use the to create the URL for the `ModelQLClient`.